python/lsst/meas/astrom/astrometry.py

```diff
@@ -313,6 +366,47 @@ def solve(self, exposure, sourceCat):
             matchMeta=matchMeta,
         )
 
+    @timeMethod
```

Comment:
Does this really need a timing decorator?

Reply:
Guess not.  Removed!